calculated_bot.py

```diff
@@ -274,9 +276,49 @@ def chunks(l, n):
     args = ctx.message.content.split(" ")
 
     if len(args) < 2:
-        await bot.send_message(ctx.message.channel,
+        PastUsers = open("UserIDs.txt", "r+")
+        NotLinked = True
+        for line in PastUsers:
+            check = line
+            check = check.strip()
+            check = check.split(":")
```

Comment:
Same thing as the other comment. Possibly put this code in a function so you don't have to write it twice.